sum/sum.py

```diff
@@ -0,0 +1,17 @@
+WORDS = ["a", "neat", "beat", "house", "me", "button", "mat"]
+
+
+def tuple_of_word_lengths():
+    """Return a tuple with the length of the shortest word,
+    the length of the longest word, and the average word length."""
+    word_lengths = ()
+
+    longest_word = max(WORDS, key=lambda x: len(x))
+    shortest_word = min(WORDS, key=lambda x: len(x))
+    average = sum(map(lambda x: len(x), longest_word + shortest_word)) // 2
+
+    word_lengths += (shortest_word, longest_word, average)
```

Comment:
One simplification is to avoid the empty tuple on line 7 and just return a tuple of the three values you obtained:

```
return (shortest_word, longest_word, average)
```